create-release-pr/README.md

```diff
@@ -0,0 +1,37 @@
+### Behavior
+
+This action will create a pull request from a release branch against `master`. It determines the release branch to create the PR for based on the current day using the following code:
+```shell
+function get_release_branch() {
+    BRANCH_DATE_STRING=$(date '+%m%d')
+    echo "release/$BRANCH_DATE_STRING"
+}
+```
+This code will look for a branch called `release/MMDD` where `MMDD` is the current month and day that the action is running. You should schedule the action to run on the day of the release. (Note the action runs in the UTC timezone)
+
```

Comment:
Not that it happens often, but for the few times the release tends to fall on a holiday, have we kept the Monday date for the release branch names? Would be good to enforce that if we haven't to ensure this action doesn't fail

Reply:
Well the downside with this action is you have to schedule it to run on one day. So right now I would plan on having it run every Monday (since cron can't follow our release schedule cleanly). This is a good point. Maybe instead of enforcing we put the wrong day in the release branch name, we just have this action look for the current day and current day + 1? 

Give it some sort of range so it can find those outliers we created without having us to lie to ourselves. Thoughts on that?

Reply:
Then we don't have to rely on us remembering, which is usually very unreliable

Reply:
I like that idea! Best of both worlds.

Reply:
@kbajno alright I have made that change. There is a new MAX_RANGE parameter where you give it a number and it will slowly creep forward that many days from the day the script is ran to look for release branches for those days.